packages/firestore/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Unreleased
+- [feature] Added an `experimentalForceLongPolling` setting that may be helpful
```

Comment:
Could we make the call to action more concrete here?

Maybe "... setting that can be used to work around proxies that prevent the Firestore client from connecting to its backend"?